### PR TITLE
IJMP-1969 Added updating of the project workspace when adding new config

### PR DESCRIPTION
**Pull request by ktsytsen**

### CODE REVIEW

File: src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt [hunk 1]
```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2020-2024 IBA Group.
```
Comment (KUGDev) [inline src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt:2]: Copyright (c) 2024 IBA Group

Comment (ktsytsen) [reply src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt]: Fixed

File: src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt
Comment (dnlsv) [inline src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt:1]: File name should be AddZoweTeamConfigActionTestSpec

Comment (ktsytsen) [reply src/test/kotlin/org/zowe/explorer/explorer/actions/AddZoweTeamConfigActionTest.kt:1]: renamed

**Review by dnlsv — APPROVED** [review]

**Pull request merged**